src/rpc/methods/eth.rs

```diff
@@ -1366,7 +1366,7 @@ pub enum EthGetBlockByHash {}
 impl RpcMethod<2> for EthGetBlockByHash {
     const NAME: &'static str = "Filecoin.EthGetBlockByHash";
     const NAME_ALIAS: Option<&'static str> = Some("eth_getBlockByHash");
-    const PARAM_NAMES: [&'static str; 2] = ["block_hash", "full_tx_info"];
+    const PARAM_NAMES: [&'static str; 2] = ["blockHash", "fullTxInfo"];
```

Comment:
I'd prefer `fullTransactionInfo`

Reply:
I think `tx` is a pretty-well known shortcut in the blockchain space, so it should be fine to leave it as is. Is it fine by you?

Reply:
I would stick with `fullTxInfo` for brevity and keep the same name as in Lotus. 'Tx' is a widely used abbreviation in the space.

Reply:
At the same time, it seems that the actual Eth API is pretty inconsistent when it comes to case.

https://www.quicknode.com/docs/ethereum/eth_getBlockByHash

It lists `transaction_detail_flag`, so `snake_case`. I could perhaps rename ours to `transactionDetailFlag`